internal/container/config.go

```diff
@@ -69,7 +69,7 @@ func parseLogLevel(s string) (endure.Level, error) {
 		return endure.DebugLevel, nil
 	case "info":
 		return endure.InfoLevel, nil
-	case "warning":
+	case "warn":
```

Comment:
```suggestion
	case "warn", "warning": // keep in mind - `warning` is an alias for `warn`
```

For the backward compatibility?

Reply:
Yeah, it makes sense.